FinancialTracker/ViewControllers/RemindersTableViewController.swift

```diff
@@ -44,19 +52,10 @@ class RemindersTableViewController: UITableViewController {
         self.transfers = transfers
     }
     
-    private func isEmpty() -> Bool {
-        for (_, value) in transfers {
-            guard value.isEmpty else {
-                return false
-            }
-        }
-        return true
-    }
-    
     // MARK: - Table view data source
     
     override func numberOfSections(in tableView: UITableView) -> Int {
-        if isEmpty() {
+        if noTransfers {
             let noDataLabel = UILabel(frame: CGRect(x: 0, y: 0, width: self.tableView.bounds.size.width, height: self.tableView.bounds.size.height))
             noDataLabel.text = "No Transfers Available"
             noDataLabel.textColor = UIColor(red: 22.0/255.0, green: 106.0/255.0, blue: 176.0/255.0, alpha: 1.0)
```

Comment:
Line 88, cell's identifier is good to be the same as the Class of the cell - `TransferTableViewCell`